src/components/wizard/wizard.component.spec.ts

```diff
@@ -401,3 +401,81 @@ describe('Wizard with validation', () => {
         return !stepElements[index].classList.contains('invalid');
     }
 });
+
+@Component({
+    selector: 'marquee-wizard-validation-appearance',
```

Comment:
There's already a text fixture for wizard with validation, I think you can reuse that.